openmeetings-db/src/main/java/org/apache/openmeetings/db/dao/calendar/AppointmentDao.java

```diff
@@ -48,6 +33,14 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.transaction.annotation.Transactional;
 
+import javax.persistence.EntityManager;
+import javax.persistence.NoResultException;
+import javax.persistence.PersistenceContext;
+import javax.persistence.TypedQuery;
+import java.util.*;
+
+import static org.apache.openmeetings.util.OpenmeetingsVariables.webAppRootKey;
+
 @Transactional
 public class AppointmentDao {
```

Comment:
Well, I don't create the ETag's the servers do, all I do is place it in the Appointment. Hence, no calculation needs to be added to AppointmentDao